IPython/extensions/autoreload.py

```diff
@@ -345,7 +345,7 @@ def __call__(self):
         return self.obj
 
 
-def superreload(module, reload=reload, old_objects={}):
+def superreload(module, reload=reload, old_objects=None):
```

Comment:
I'm not yet sure whether or not this one was deliberate. The autoreload code is rather... interesting.